pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -138,7 +154,7 @@ private static int copyStream(InputStream inputStream, OutputStream outputStream
   }
 
   private final @Nullable InputStream stream;
-  private @Nullable StreamWrapperFinalizeAction finalizeAction;
+  private LazyCleaner.CleaningAction finalizeAction;
```

Comment:
We do not need to hold a reference to `finalizeAction`. We need to hold a reference to `cleanable` only, so we call its `.clean()` in our `.close()`

Reply:
we need it for getStream https://github.com/pgjdbc/pgjdbc/blob/3e2db5ff4c3da3240e21129bc3e7ddc389fba9b7/pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java#L108
If we don't need that code then we can get rid of the reference.